src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const listElement = document.querySelector('ul');
+
+const parseSalary = (salary) => {
+  const parsedSalary = salary.replace('$', '').replace(',', '');
+
+  try {
+    return Number(parsedSalary);
+  } catch (error) {
+    return 0;
+  }
+};
```

Comment:
why do we need this `try/catch` block? 
```suggestion
const parseSalary = (salary = '') => {
  const parsedSalary = salary.replace('$', '').replace(',', '');

  return +parsedSalary;
};
```